.travis.yml

```diff
@@ -4,5 +4,10 @@ python:
   - "2.7"
   - "3.3"
   - "3.4"
-install: "pip install -r requirements.txt"
-script: py.test 
+install:
+  - pip install -r requirements.txt
+  - pip install codecov pytest-cov
```

Comment:
@stevepeak, just curious: why not just add `codecov` and `pytest-cov` to requirements.txt?